lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -261,10 +261,11 @@ defmodule ExUnit.DocTest do
           reraise e, stack
```

Comment:
I assume this is here because in the few places elsewhere an `ExUnit.AssertionError` is rescued it is reraised instead of creating a new `ExUnit.AssertionError`. We should be safe to remove this as the new exception will contain the previous exceptions message.